00_Base/src/services/TransactionFilterService.ts

```diff
@@ -56,18 +56,13 @@ export class TransactionFilterService implements ITransactionDatasource {
 
     const [transactions, total] = await Promise.all([
       this.transactionRepository.transaction.readAllByQuery(queryOptions),
-      Transaction.count(
-        countQueryOptions as Omit<
-          CountOptions<Attributes<Transaction>>,
-          'group'
-        >,
-      ),
+      Transaction.count(countQueryOptions),
     ]);
 
     const result: PaginatedResult<Transaction> =
       new PaginatedResult<Transaction>();
     result.data = transactions;
-    result.total = total;
+    result.total = total[0].count;
```

Comment:
If the omit group part is removed, do we need to get a sum of the `total` array like below? or total[0] is always the actual total number?
```suggestion
    if (total.length === 1) {
      result.total = total[0].count;
    } else {
      result.total = total.reduce((sum, group) => (sum + group.count), 0);
    }
```